packages/contracts-bedrock/test/L2/CrossL2Inbox.t.sol

```diff
@@ -101,7 +101,7 @@ contract CrossL2InboxTest is Test {
         vm.expectCall({ callee: _target, msgValue: _value, data: _message });
 
         // Look for the emit ExecutingMessage event
-        vm.expectEmit(Predeploys.CROSS_L2_INBOX);
+        vm.expectEmit(true, true, true, true, Predeploys.CROSS_L2_INBOX);
```

Comment:
small nit: just using `vm.expectEmit(address)` is the same as `vm.expectEmit(true,true,true,true,address(0x..)`. We prefer the first because it is less code